We'd like to use the hardcoded configurations for mesa. Upstream has added an option `xmlconfig`.

--- a/src/util/xmlconfig.h
+++ b/src/util/xmlconfig.h
@@ -39,7 +39,7 @@
 extern "C" {
 #endif
 
-#if defined(ANDROID) || defined(_WIN32)
+#if 1
 #define WITH_XMLCONFIG 0
 #else
 #define WITH_XMLCONFIG 1
